project/AppDependencies.scala

```diff
@@ -8,16 +8,16 @@ object AppDependencies {
   }
 
   private val compile = Seq(
-    "uk.gov.hmrc"              %% "bootstrap-frontend-play-27" % "2.25.0",
-    "uk.gov.hmrc"              %% "govuk-template"             % "5.57.0-play-27",
-    "uk.gov.hmrc"              %% "play-ui"                    % "8.18.0-play-27",
+    "uk.gov.hmrc"              %% "bootstrap-frontend-play-27" % "3.4.0",
+    "uk.gov.hmrc"              %% "govuk-template"             % "5.61.0-play-27",
+    "uk.gov.hmrc"              %% "play-ui"                    % "8.21.0-play-27",
     "uk.gov.hmrc"              %% "url-builder"                % "3.4.0-play-27",
-    "uk.gov.hmrc"              %% "auth-client"                % "3.2.0-play-27",
-    "uk.gov.hmrc"              %% "play-language"              % "4.4.0-play-27",
+    "uk.gov.hmrc"              %% "auth-client"                % "3.3.0-play-27",
+    "uk.gov.hmrc"              %% "play-language"              % "4.10.0-play-27",
     "commons-validator"         % "commons-validator"          % "1.6",
     "org.apache.httpcomponents" % "httpclient"                 % "4.4.1",
-    "uk.gov.hmrc"              %% "play-frontend-govuk"        % "0.56.0-play-27",
-    "uk.gov.hmrc"              %% "play-frontend-hmrc"         % "0.32.0-play-27"
+    "uk.gov.hmrc"              %% "play-frontend-govuk"        % "0.60.0-play-27",
```

Comment:
Be interesting to know what would happen if we remove the `play-frontend-govuk` dependency, to see if everything is available transitively without having to then worry about compatible versions.